client/config/config.go

```diff
@@ -92,220 +48,104 @@ func Cmd(defaultCLIHome string) *cobra.Command {
 
 	cmd.Flags().String(flags.FlagHome, defaultCLIHome,
 		"set client's home directory for configuration")
-	cmd.Flags().Bool(flagGet, false,
-		"print configuration value or its default if unset")
+	//	cmd.Flags().Bool(flagGet, false,
+	//		"print configuration value or its default if unset")
 	return cmd
 }
 
 func runConfigCmd(cmd *cobra.Command, args []string) error {
 
-	
-	InitConfigTemplate()
+	v := viper.New()
 
-	cfgPath, err := ensureConfFile(viper.GetString(flags.FlagHome))
+	cfgPath, err := ensureCfgPath(v.GetString(flags.FlagHome))
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to make config path: %v\n", err)
 		return err
 	}
 
-	getAction := viper.GetBool(flagGet)
-	if getAction && len(args) != 1 {
-		return ErrWrongNumberOfArgs
-	}
-
-	cliConfig, err := getClientConfig(cfgPath)
+	cliConfig, err := getClientConfig(cfgPath, v)
 	if err != nil {
-		return fmt.Errorf("Unable to get client config %v", err)
+		fmt.Fprintf(os.Stderr, "Unable to get client config: %v\n", err)
+		return err
```

Comment:
@alessio  Following your suggestion, I print an error here to os.Stderr. I am unable to add os.Exit(1) cause the runConfigCmd is supposed to return an error.